docs/Tutorials/Visualization.md

```diff
@@ -50,9 +50,9 @@ files. An XDMF file must be created from the volume data in order to do
 visualization using ParaView. To this end we provide the tool
 `generate-xdmf` in the Python command-line interface. Run it in your build
 directory as `spectre generate-xdmf`. It
-takes three required arguments which are passed to `--file-prefix`,
-`--subfile-name`, and `--output`. The argument passed to `--file-prefix` is the
-name of the H5 volume data, leaving out the node number and extension. The
+takes two required arguments which are passed to `--subfile-name`, and
+`--output` (for more detail on the options, use `generate-xdmf --help`).
```

Comment:
This is actually written below already, so just delete the part in parentheses (sorry haven't noticed it before)

Reply:
Second push removes that sentence.